indexing-service/src/main/resources/indexer_static/js/console-0.0.1.js

```diff
@@ -2,39 +2,56 @@
 
 var oTable = [];
 
+var killTask = function(taskId) {
+  if(confirm('Do you really want to kill: '+taskId)) {
+    $.ajax({
+      type:'POST',
+      url: '/druid/indexer/v1/task/'+ taskId +'/shutdown',
+      data: ''
+    }).done(function(data) {
+      alert('Submitted kill request for: '+data.task);
```

Comment:
once the task is killed the page needs to be refreshed to see the killed tasks in completed task list...that's why added this alert to give user a confirmation. This is actually the current behavior of overlord console, even if task completes normally you will need to refresh the page to see that task in completed list